client/cli/src/main.rs

```diff
@@ -542,9 +542,24 @@ async fn handle_command(
                 .iter()
                 .map(|(amount, notes)| (amount.to_owned(), notes.len()))
                 .collect();
+
+            let federation_name = client
+                .config()
+                .as_ref()
+                .meta
+                .get(META_FEDERATION_NAME_KEY)
+                .ok_or_else(|| {
```

Comment:
If it's required... why make it an "additional config value"?

Reply:
I think it was a suggestion by @justinmoon to make it not required anymore. There could be a BIP-like process to define meta-data extensions. And even though most federations will have a name, it's not *really* necessary.

Reply:
Makes sense, but then we should remove this error here? Or default to "unnamed" for now?

Reply:
Right, the client needs to be able to just display all the meta fields.

Reply:
Done.